2 Niagara Fundamentals Exercise - Multiplexer/multiplexer-rt/src/com/tridiumuniversity/multiplexer/BMultiplexer.java

```diff
@@ -0,0 +1,257 @@
+package com.tridiumuniversity.multiplexer;
+
+import javax.baja.nre.annotations.NiagaraAction;
+import javax.baja.nre.annotations.NiagaraProperty;
+import javax.baja.nre.annotations.NiagaraType;
+import javax.baja.sys.*;
+import javax.baja.sys.BBoolean;
+
+import javax.baja.sys.BComponent;
+import javax.baja.sys.Flags;
+
+@NiagaraType
+
+@NiagaraProperty(
+    name = "ln0",
```

Comment:
I think you may have used an "L" here rather than an "i". I meant for these to be named "in0", "in1", etc.